### PR TITLE
PEP8 Fixes for topic_coherence and corpora

### PR DESCRIPTION
**Pull request by souravsingh**
Fixes a part of #965

### CODE REVIEW

**Comment by souravsingh** [comment]
I have ignore all E501 PEP8 errors in the PR

**Comment by souravsingh** [comment]
@piskvorky Pull request for some PEP8 fixes are done here.

File: gensim/corpora/dictionary.py [hunk 1]
```diff
@@ -24,13 +24,12 @@
...
 from gensim import utils
 
-if sys.version_info[0] >= 3:
-    unicode = str
-
 from six import PY3, iteritems, iterkeys, itervalues, string_types
 from six.moves import xrange
 from six.moves import zip as izip
 
+if sys.version_info[0] >= 3:
```
Comment (tmylk) [inline gensim/corpora/dictionary.py]: What is the reason for this line?

Comment (souravsingh) [inline gensim/corpora/dictionary.py]: I have no clue about the line...

File: gensim/corpora/lowcorpus.py [hunk 2]
```diff
@@ -166,7 +166,7 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
 
         if truncated:
             logger.warning("List-of-words format can only save vectors with "
-                            "integer elements; %i float entries were truncated to integer value" %
+                            "integer elements; %i float entries were truncated to integer value"
```
Comment (tmylk) [inline gensim/corpora/lowcorpus.py]: the '%' sign should stay there for the substitution to work. Why was it removed?

File: gensim/corpora/sharded_corpus.py [hunk 3]
```diff
@@ -26,6 +26,13 @@
 import scipy.sparse as sparse
 import time
 
+from six.moves import xrange
+
+import gensim
```
Comment (tmylk) [inline gensim/corpora/sharded_corpus.py]: gensim imports should be after all the other imports

File: gensim/corpora/dictionary.py (hunk 1 above)
Comment (tmylk) [inline gensim/corpora/dictionary.py]: this line should be where it was before.

As per PEP8:

Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports

You should put a blank line between each group of imports.

File: gensim/corpora/sharded_corpus.py (hunk 3 above)
Comment (tmylk) [inline gensim/corpora/sharded_corpus.py]: Specifically after Theano. 

See pep8 for order of imports

**Comment by piskvorky** [comment]
See my review comments from all your other (now closed) PRs.

**Comment by souravsingh** [comment]
I have addressed all the comments made by @tmylk. Let me know if there are more.

**Comment by tmylk** [comment]
Thanks for the PR!

**Comment by piskvorky** [comment]
None of my review comments were addressed. Please revert @tmylk .

**Comment by tmylk** [comment]
Reverted.

**Pull request merged**